diff --git a/front-end/src/app/components/project/project-create/project-create.component.ts b/front-end/src/app/components/project/project-create/project-create.component.ts
index 7b2b55e..fc13da8 100644
--- a/front-end/src/app/components/project/project-create/project-create.component.ts
+++ b/front-end/src/app/components/project/project-create/project-create.component.ts
@@ -1,11 +1,63 @@
+import { CommonModule } from '@angular/common';
 import { Component } from '@angular/core';
-
+import { FormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { ProjectServiceService } from '../service/project-service.service';
+interface Project {
+  id: string;
+  name: string;
+  department: string;
+  employees: string[];
+  customer: string;
+  requirements: string;
+  skills: string;
+  result_image: string[];
+  startDate: string;
+  endDate: string;
+  status: string;
+}
 @Component({
   selector: 'app-project-create',
-  imports: [],
+  standalone: true,
+  imports: [CommonModule,FormsModule,RouterModule],
   templateUrl: './project-create.component.html',
   styleUrl: './project-create.component.scss'
 })
 export class ProjectCreateComponent {
+  projects: Project[] = [];
+  newProject : Project = {
+    id: '',
+    name: '',
+    department: '',
+    employees: [],
+    customer: '',
+    requirements: '',
+    skills: '',
+    result_image: [],
+    startDate: '',
+    endDate: '',
+    status: ''
+  }
+  constructor(private projectService: ProjectServiceService) {}
 
+  addProject():void {
+    this.projectService.createProject(this.newProject).subscribe(
+      (project) => {
+        this.projects.push(project)
+        this.newProject = {
+          id: '',
+          name: '',
+          department: '',
+          employees: [],
+          customer: '',
+          requirements: '',
+          skills: '',
+          result_image: [],
+          startDate: '',
+          endDate: '',
+          status: ''
+        }
+      }
+    )
+  }
 }
diff --git a/front-end/src/app/components/project/project-index/project-index.component.ts b/front-end/src/app/components/project/project-index/project-index.component.ts
index e2f84a0..71cb293 100644
--- a/front-end/src/app/components/project/project-index/project-index.component.ts
+++ b/front-end/src/app/components/project/project-index/project-index.component.ts
@@ -1,11 +1,36 @@
-import { Component } from '@angular/core';
-
+import { Component, OnInit } from '@angular/core';
+import { ProjectServiceService } from '../service/project-service.service';
+interface Project {
+  id: string;
+  name: string;
+  department: string;
+  employees: string[];
+  customer: string;
+  requirements: string;
+  skills: string;
+  result_image: string[];
+  startDate: string;
+  endDate: string;
+  status: string;
+}
 @Component({
   selector: 'app-project-index',
   imports: [],
   templateUrl: './project-index.component.html',
   styleUrl: './project-index.component.scss'
 })
-export class ProjectIndexComponent {
+export class ProjectIndexComponent implements OnInit {
+
+  projects: Project[] = []
+  constructor(private projectService: ProjectServiceService){}
+
+  ngOnInit(): void {
+      this.projectService.getProjecst().subscribe(
+        (data) => {
+          this.projects = data
+        }
+      )
+  }
+
 
 }
